code/modules/spells/spell_types/pointed/tie_shoes.dm

```diff
@@ -0,0 +1,130 @@
+
+/datum/action/cooldown/spell/pointed/untie_shoes
+	name = "Untie Shoes"
+	desc = "This unassuming spell unties and then knots the target's shoes."
+	ranged_mousepointer = 'icons/effects/mouse_pointers/lace.dmi'
+	button_icon_state = "lace"
+
+	school = SCHOOL_CONJURATION
+	cooldown_time = 3 SECONDS
+	cooldown_reduction_per_rank = 0.2 SECONDS
+
+	spell_max_level = 4
+	invocation = "Acetato!"
```

Comment:
me myself personally i would name this "Agaletto!" because it sounds like "Aglets" (the little plastic nibs at the end of shoelaces) but it's you call. just thought i'd mention it so ignore at your convenience.

Reply:
added as a chance